nixos/maintainers/scripts/lxd/lxd-virtual-machine-image-inner.nix

```diff
@@ -14,7 +14,18 @@
     ];
 
   networking.useDHCP = false;
-  networking.interfaces.eth0.useDHCP = true;
+
+  systemd.network = {
```

Comment:
we should also set
```
  networking.useNetworkd = true;
  networking.dhcpcd.enable = false;
  services.resolved.dnssec = "false";
```

Reply:
No, we should not `useNetworkd`, which maps `networking.interfaces` options to `networkd`. We should instead use `systemd.networkd` options directly, because it is less surprising.

DNSSEC validation is [disabled by default](https://search.nixos.org/options?channel=unstable&show=services.resolved.dnssec&from=0&size=50&sort=relevance&type=packages&query=resolved) on resolved.

Reply:
Is networking.dhcpcd disabled by default if no useDHCP is used?

Reply:
Why does  `services.resolved.dnssec` need to be disabled?

Reply:
dnssec enabled as a default is quite problematic, since in regular networks you often don't get a fully dnssec compliant dns resolver. dns hijacking (for captive portals, etc) and simply missing/partial dnssec are common and resolved will refuse to use the resolver if it detects anything like that, therefore breaking dns. given TLS exists the security argument here doesn't weigh too much.

Reply:
For reference 4f461f7b7788d9aafc6021f1384423dabe605ad0